### ISSUE TITLE
Conflict with phyloseq?

### ISSUE THREAD

**Issue by michoug** [YuLab-SMU/tidytree#10]
Hi,
When I load tidytree (v0.2.6) and phyloseq (v1.28.0) together and use the phyloseq function `read_tree` as in this example

```
library(phyloseq)
library(tidytree)

treefilename <- read_tree(system.file("extdata", "esophagus.tree.gz", package="phyloseq"))
```

I get these warnings

```
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
```

Any ideas?

**Comment by GuangchuangYu** [comment]
you can just ignore it.

**Comment by xvazquezc** [comment]
First at all my apologies for posting into a closed issue, but I don't think I should open a new one about the same topic. I wouldn't bother if the messages appeared a couple of times, but whenever a function runs on the phyloseq object, esp. those that iterate through it, it outputs the message.

I've been trying to knit a Rmarkdown report with this issue and it becomes impossible to make it readable at all or find anything: more than half of the document is made of those messages and they can't be silenced even with `warnings = FALSE, error = FALSE` (in `knitr`).

Using phyloseq v1.30.0 and tidytree 0.3.3

**Comment by cjfields** [comment]
@xvazquezc just curious, but was this within a R Markdown report with chunks?  I am seeing this too, and I think it also causes issues with plots (I am getting a NULL instance back with this so am resorting to using `phyloseq` and `plot_tree`).  I do not see this when running `ggtree` interactively.

**Comment by xvazquezc** [comment]
@cjfields yes, the RMarkdown report had chunks, but I also get the messages in the RStudio console if I run the code interactively. It happens any time a `phylo` object is called. I haven't use `ggtree` with `phylo` objects for a while so I don't know if it happened too

**Comment by jgrembi** [comment]
@xvazquezc @cjfields @michoug I ran into the same issue and finally got the messages to stop by adding `message = FALSE` in the code chunk headers. This worked for both the RStudio interactive console and the Rmarkdown (both PDF and HTML output formats).  Hope it works for you too!

**Comment by harper357** [comment]
This still seems to be a problem and I don't think "just ignore it" is a solution. There doesn't seem to be a way to turn the warning off and it is output more than 100 time in several of my scripts making the outputs very hard to parse.

**Comment by BellaTsch** [comment]
Same problem here...

Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘RNeXML’ ‘tidytree’

Anyone has a solution?

**Comment by bengouts** [comment]
Hey,  
I got the same anoying problem. I solved that using `options(getClass.msg=FALSE)` that I found [here](https://github.com/epurdom/clusterExperiment/issues/66#issuecomment-214519899)
Hope it helps :)

**Comment by dansmith01** [comment]
Still an issue.
```
Found more than one class "phylo" in cache; using the first, from namespace 'phyloseq'
Also defined by ‘tidytree’
```
I'd like to use tidytree in my own CRAN package, but don't want to bombard users with these error messages.

**Comment by harper357** [comment]
@dansmith01 You may have to open a new issue, this one has been closed for more than 4 years

**Comment by ashishdamania** [comment]
One workaround is to load the Phyloseq package at the very last so that it can take the precedence over tidytree package.
